src/component/NMRium.tsx

```diff
@@ -455,12 +455,14 @@ export default memo(NMRium);
 function StateError() {
   const { errorAction } = useChartData();
   const { error: alertError } = useAlert();
+  const { logger } = useLogger();
 
   useEffect(() => {
     if (!errorAction) return;
 
     alertError(errorAction?.message ?? String(errorAction));
```

Comment:
No need to use the alert because the logger modal will popup automatically if the log level is warning, error, or fatal

Reply:
use logger instead alert done